acceptance/.beaker.yml

```diff
@@ -0,0 +1,15 @@
+---
+ssh:
+  keys:
+    - id_rsa_acceptance
+    - ~/.ssh/id_rsa-acceptance
+xml:                         true
+timesync:                    false
+repo_proxy:                  true
+add_el_extras:               false
+forge_host:                  forge-aio01-petest.puppetlabs.com
+'master-start-curl-retries': 30
+log_level:                   debug
+preserve_hosts:              onfail
+helper:                      ./lib/helper.rb
```

Comment:
I noticed in the old common options, it has `:load_path                   => './lib',`. Is there a reason you decided to use this rather then setting `:load_path` here?

Reply:
It turned out the `helper` updates the load_path in the same way, so specifying `:load_path` was redundant. Also using the helper allows us to do more things in the future (if need be).